Providers/nxOMSAutomationWorker/automationworker/worker/runtime.py

```diff
@@ -174,4 +177,4 @@ def get_default_python_interpreter_major_version():
     if p.returncode == 0:
         return int(default_interpreter_version.strip())
     else:
-        return None
\ No newline at end of file
+        return None
```

Comment:
is it a new line added ?

Reply:
I will remove it

Reply:
I tried removing it, but it seems to be added automatically